### ISSUE TITLE
EmptyLinesAroundBody & EmptyLinesAroundAccessModifier fight each other

### ISSUE THREAD

**Issue by andreaseger** [rubocop/rubocop#1287]
take a simple example like this

``` ruby
class Foo
  def some_method
    "foo"
  end

  private
end
```

using rubocop `0.25.0` the following command seems to go into an endless loop.

``` sh
rubocop --auto-correct --only EmptyLinesAroundBody,EmptyLinesAroundAccessModifier 
empty_lines_around_access_modifier_fail.rb
```

I guess one correction adds a empty line where the other removes it again.

You might say that this AccessModifier is useless anyway and we have a Cop for that named `Lint/UselessAccessModifier`. The problem is this Cop does not (yet) support auto-correct.

I could not find it on [this](https://github.com/bbatsov/rubocop/wiki/Automatic-Corrections) list on the wiki and to me it seems both possible and save to do auto-correct.

If someone can point me to a good example on how to do this kind of auto-correction or how the auto-correction is applied in general I probably could make a pull request myself.
